### ISSUE TITLE
sd_bus_add_object_vtable man page. Note about making reply asynchronus should be in separated paragraph.

### ISSUE THREAD

**Issue by igo95862** [systemd/systemd#17626]
```
When a request is received, any associated callbacks are called sequentially until a callback returns a non-zero integer. 
Return zero from a callback to give other callbacks the chance to process the request. Callbacks are called in the following
 order: first,  callbacks installed with sd_bus_add_filter() are called. Second, callbacks attached directly to the request 
object path are  followed by any D-Bus method callbacks attached to the request object path, interface and member. 
Finally, the property  attached to the request object path, interface and member are called. If the final callback returns 
zero, an error reply is sent back the caller indicating no matching object for the request was found. Note that you can 
return a positive integer from a callback  immediately sending a reply. This informs sd-bus this callback will take 
responsibility for replying to the request without forcing callback to produce a reply immediately. This allows a callback to 
perform any number of asynchronous operations required to construct reply. Note that if producing a reply takes too 
long, the method call will time out at the caller.
```

Right now it buried inside a big paragraph that has little to do with it. I think its kind of important trick. I wanted to open a request to add an ability to make replies asynchronous but made a small experiment with returning positive number. Only after I discovered that it was mentioned in the man page. I think that's an man page failure.

**Comment by DaanDeMeyer** [comment]
My bad, I wrote that but didn't really split it up correctly. Are you willing to make a PR for this? You can just edit the corresponding xml file and make a PR. I'll be happy to review it.

**Comment by igo95862** [comment]
Sure

**Comment by igo95862** [comment]
I also changed the second 'Note' to 'However, '
